DiscordBee.cs

```diff
@@ -230,29 +228,18 @@ string padString(string input)
       // Button Functionality
       if (_settings.ShowButton)
       {
-        var uri = _settings.ButtonUrl
-          .Split('/')
-          .Select(part =>
-          {
-            var result = _layoutHandler.Render(part, metaDataDict, _settings.Seperator);
-            if (part != result && (part != "_"))
-            {
-              return WebUtility.UrlEncode(result);
-            }
-
-            return part;
-          });
+        var uri = _layoutHandler.RenderUrl(_settings.ButtonUrl, metaDataDict, _settings.Seperator);
```

Comment:
I think separators should be empty in this case because some of those characters might be part of the userdefined URL and then they would be stripped and therefore the URL would be malformed in a non expected way.

Reply:
Oh yeah I forgot. Anyways, I changed it on the latest commit.